helm/values.yaml

```diff
@@ -0,0 +1,43 @@
+---
+replicaCount: 1
+securityContext:
+  fsGroup: 123
+container:
+  image: jmatejka/happy-bank-core
+  tag: 0.2.1
+  containerPort: 5000
+  volumeMountPath: /data
+  healthEndpoint: /health
+storageClass:
+  provisioner: kubernetes.io/no-provisioner
+  volumeBindingMode: WaitForFirstConsumer
+persistentVol:
+  accessMode: ReadWriteOnce
+  capacity: 1Gi
+  localPath: /data
+  node: minikube
+  reclaimPolicy: Retain
+  storageClass: local-storage
+persistentVolClaim:
+  accessMode: ReadWriteOnce
+  capacity: 1Gi
+  storageClass: local-storage
+service:
+  protocol: TCP
+  type: ClusterIP
+  port: 5000
+ingress:
+  host: core.happybank.io
+  pathType: Prefix
+job:
+  completions: 1
+  backoffLimit: 2
+  containerName: curl-container
+  image: appropriate/curl
+  restartPolicy: OnFailure
+secret:
+  type: Opaque
+  token: PENPUkVfVE9LRU4+
+config:
+  host: 127.0.0.1
+  port: 6379
```

Comment:
1. It's tough to be oriented on the first brief look. Spaces between dictionaries would help a lot.
2. To avoid 'repeating yourself' = code duplication is better to arrange the values (configuration) structure by the microservices view. It means, if we have core app, its verification job and preparation for Redis DB at this point, we should split the config by them. Then, it is easier to maintain as well.